geode-docs/developing/expiration/configuring_data_expiration.html.md.erb

```diff
@@ -21,14 +21,14 @@ limitations under the License.
 
 Configure the type of expiration and the expiration action to use.
 
-<a id="configuring_data_expiration__section_ADB8302125624E01A808EA5E4FF79A5C"></a>
+-   Set the region's `statistics-enabled` attribute to `true`.
```

Comment:
I recommend moving this item down to after setting the expiration type.
I would then add something like "If idle-time expiration is used then set the region's 'statistics-enabled' attribute to 'true'. Idle-time expiration uses CacheStatistics.getLastAccessedTime which is only available if the region's statistics are enabled.

Reply:
We placed it at the beginning because the expiration requires the statistics-enabled to be set regardless of the type of expiration (at least through GFSH command), even though they may only be required for idle-time expiration.